libs/tokens/src/pure/TokenLogo/index.tsx

```diff
@@ -207,19 +102,31 @@ export function TokenLogo({ logoURI, token, className, size = 36, sizeMobile, no
     <Slash />
   )
 
-  if (noWrap) return tokenContent
+  if (noWrap) return actualTokenContent
 
   const chainInfo: BaseChainInfo | undefined = getChainInfo(token?.chainId as SupportedChainId)
   const chainName = chainInfo?.label || ''
 
+  // This is the size of the chain logo
+  const chainLogoSizeForCalc = size / 2.2
```

Comment:
It's better to move 2.2 to a const with description (why 2.2?)

Reply:
Addressed.